src/client-side-encryption/client_encryption.ts

```diff
@@ -866,9 +866,35 @@ export interface ClientEncryptionOptions {
    */
   tlsOptions?: CSFLEKMSTlsOptions;
 
-  /** @internal TODO(NODE-5688): make this public
+  /**
    *
    * The timeout setting to be used for all the operations on ClientEncryption.
+   *
+   * When provided, `timeoutMS` is used as the timeout for each operation executed on
+   * the ClientEncryption object.  For example:
+   *
+   * ```typescript
+   * const clientEncryption = new ClientEncryption(client, {
+   *  timeoutMS: 1_000
+   *  kmsProviders: { local: { key: '<KEY>' } }
+   * });
+   *
+   * // `1_000` is used as the timeout for createDataKey call
+   * await clientEncryption.createDataKey('local');
+   * ```
+   *
+   * If `timeoutMS` is configured on the provided client, the client's timeoutMS value
+   * will be used unless `timeoutMS` is also provided as a separate option.
```

Comment:
```suggestion
   * will be used unless `ClientEncryptionOptions.timeoutMS` is also provided as a separate option.
```
This is a bit clearer to me considering we're using `timeoutMS` to refer to both `ClientEncryptionOptions.timeoutMS` and `MongoClient.options.timeoutMS`